executor/builder.go

```diff
@@ -2164,7 +2164,12 @@ func (b *executorBuilder) buildAnalyzeIndexIncremental(task plannercore.AnalyzeI
 }
 
 func (b *executorBuilder) buildAnalyzeSamplingPushdown(task plannercore.AnalyzeColumnsTask, opts map[ast.AnalyzeOptionType]uint64, autoAnalyze string, schemaForVirtualColEval *expression.Schema) *analyzeTask {
-	job := &statistics.AnalyzeJob{DBName: task.DBName, TableName: task.TableName, PartitionName: task.PartitionName, JobInfo: autoAnalyze + "analyze table"}
+	colStrs := make([]string, 0, len(task.ColsInfo))
+	for _, col := range task.ColsInfo {
+		colStrs = append(colStrs, col.Name.L)
+	}
```

Comment:
`task.ColsInfo` contains all the columns when analyze all columns. So here we want to list all columns which need to be analyzed for all cases rather than list the columns only for analyze predicate columns or analyze column c1, ..., cn?